content/hilfe/Handbuch/Einstellungen/dokumente/dokumente.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Einstellungen
+weight: 8
+icon: book
+keywords:
+- einstellungen, dokumente, kategorien
+
+---
+
+Hier finden Sie verschiedene Optionen, die die Verwaltung ihrer Dokumente betreffen. Denn mit einem gut geführten digitalen Dokumentenarchiv sparen Sie sich Zeit, Stress und Papier und machen sich das teilen von Dateien einfacher. 
+
+### Kategorien
+
+Hier können Sie die farbliche Codierung ihrer **Dokumentenkategorien** individuell anpassen oder neue Kategorien erstellen. 
```

Comment:
Ich nehme an hier kommt noch mehr? Die Datei müsstest du außerdem in `index.md` umbenennen, sonst taucht der Artikeln nicht auf.